### PR TITLE
weggli 0.2.3 (new formula)

### PR DESCRIPTION
**Pull request by fabianfreyer**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/weggli.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+class Weggli < Formula
+  desc "Is a fast and robust semantic search tool for C and C++ codebases"
```
Comment (nandahkrishna) [inline Formula/weggli.rb:2]:
```suggestion
  desc "Fast and robust semantic search tool for C and C++ codebases"
```

File: Formula/weggli.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  homepage "https://github.com/googleprojectzero/weggli"
+  url "https://github.com/googleprojectzero/weggli/archive/refs/tags/v0.2.3.tar.gz"
+  sha256 "240ccf2914533d7c2114901792240b95a6b632432ddd91d1fe3bbb060e010322"
+  license "Apache-2.0"
+  head "https://github.com/googleprojectzero/weggli.git", branch: "main"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
```
Comment (nandahkrishna) [inline Formula/weggli.rb:12]: We don't need this by default.

**Review by nandahkrishna — COMMENTED** [review]
Thanks for your PR, @fabianfreyer. I have a couple of suggestions.

**Review by carlocab — APPROVED** [review]
Thanks, @fabianfreyer.

**Pull request closed without merging**